src/main/java/com/example/petbuddybackend/repository/rating/RatingRepository.java

```diff
@@ -4,9 +4,17 @@
 import org.springframework.data.domain.Page;
 import org.springframework.data.domain.Pageable;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 
 
 public interface RatingRepository extends JpaRepository<Rating, Long> {
 
     Page<Rating> findAllByCare_Caretaker_Email(String caretakerEmail, Pageable pageable);
+
+    @Query("""
+        SELECT AVG(r.rating)
+        FROM Rating r
+        """)
+    Float getAvgRating();
```

Comment:
```suggestion

    @Query("SELECT COALESCE(AVG(r.rating), 0.0) FROM Rating r")
    Float getAvgRating();
```
Ja byłbym chyba za tym żeby 0 zwracać jak nie ma ocen. Wiem że to edge case, ale jak mówi prawo Murphiego: jak coś się może zepsuć to się zepsuje xd